tasks/stage-product/task.sh

```diff
@@ -24,7 +24,7 @@ function main() {
 
   local version
   pushd "${cwd}/pivnet-product"
-    version="$(ls -1 *.pivotal | sed "s/"${PRODUCT_NAME}"-\(.*\).pivotal/\1/")"
+    version="$(unzip -p *.pivotal 'metadata/*.yml' | cat | grep 'product_version:' | cut -d ':' -f 2 | tr -d ' ')"
```

Comment:
The other code update: 
```{"Key":"custom_hostname","Value":"$OPSMAN_NAME"}```
is not related to this PR and I meant to create a separate PR. Let me know if you want it in a separate PR and I can do it.
Basically, it was a code change tried and verified by a customer to customize the Ops Mgr VM custom hostname, in order to differentiate the vm name from the default one in the .ovf.